### ISSUE TITLE
Any chance of a more recent release?

### ISSUE THREAD

**Issue by Vaelatern** [ArchipelProject/Archipel#1182]
Thank you!

**Comment by phillipp** [comment]
I would second this. I think it may be a good point right now to do some polish and release a stable version. The project looks pretty abandoned right now and if it would look a little more active (and this inclused releases as this is the thing a lot of people look first when having a glance at a project) with new releases.

Any ideas from the maintainers?

**Comment by ikus060** [comment]
Well, the project seam dead. Last release, beta 6, was publish in March 2013. And last activities in git is 6 month old. It's really sad, archipel seam to have a good potential.

**Comment by phillipp** [comment]
@ikus060 not sure, because when I tried archipel I could not find a bug and the project seems feature complete. It may be just done and working fine.... It's possible that it only *looks* abandoned but isn't.

**Comment by ikus060** [comment]
Then will see how long it take for a maintainer to reply to this issue...

**Comment by CyrilPeponnet** [comment]
It's not abandoned and I'm using it on nearly 400 hypervisors. Not features complete either but hard to find the time and the people to contribute to.

**Comment by ikus060** [comment]
It's really nice to get a reply! Back to the issue. Any chance to get a new release?

**Comment by CyrilPeponnet** [comment]
Master is the release ;)

**Comment by phillipp** [comment]
@CyrilPeponnet I think one large problem is, that the project just looks dead. There is no recent release (regardless of what changed) and there isn't even a "THIS PROJECT IS NOT DEAD" info in the Readme. If you want to get for contributors, you need someone who uses this project first. But this won't happen with this strategy, I think. So again, I'd propose tagging a release so at least peoply try this project out. And only THEN you can get contributors. Tagging a release takes a few minutes but will be a game changer for how the project looks like regarding the maintainer situation.

**Comment by K1773R** [comment]
yes master is, but the python build via pip is horrible outdated and most will use that and be disappointed.

**Comment by phillipp** [comment]
Maybe we just open a pull request with a new package definition so everybody can install the current master (best would be with a git tag) from pip/pypi? unfortunately I have absolutely no idea how this works in python (ruby dev myself).

**Comment by paradisaeidae** [comment]
Agree with phillipp. Any chance of ripping out dependencies on python, replace with dependencies of Ruby?

**Comment by ikus060** [comment]
If you look at git history you will notice this project is dead. Latest commit is years old. I really recommend you to use a different product to manage your VM. The best one I found is Proxmox. It really worth a try and provide business grade feature like flexible storage, clustering, etc.
